.github/workflows/whylogs-ci.yml

```diff
@@ -85,8 +85,21 @@ jobs:
       - name: Run pre-commit hooks
         run: make pre-commit
 
+      - run: |
+          wget https://sqlite.org/2022/sqlite-autoconf-3380500.tar.gz
+          tar -xvf sqlite-autoconf-3380500.tar.gz
+      - run: |
+          cd sqlite-autoconf-3380500
+          ./configure
+          make
+          sudo make install
+          export PATH="/usr/local/lib:$PATH"
+          cd ..
```

Comment:
Would be better to have an image that just has sqlite, or extend one that we want and install sqlite ontop of it